src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -987,4 +989,86 @@ public static void replaceValueInInputPropertyFromEnv(String pluginProperty, Str
       replaceValueInInputProperty(pluginProperty, envVariableKey);
     }
   }
+
+  /**
+   * Click on the Page Navigation button inside Plugin's properties page
+   *
+   * @param button ButtonName
+   */
+  public static void clickOnPageButton(String button) {
+    WaitHelper.waitForElementToBeClickable(CdfPluginPropertiesLocators.locateButton(button),
+                                           ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
+    ElementHelper.clickOnElement(CdfPluginPropertiesLocators.locateButton(button));
+  }
+
+  /**
+   * Select Source Plugin from the Replication list
+   *
+   * @param pluginName PluginName
+   */
+  public static void selectReplicationSourcePlugin(String pluginName) {
+    ElementHelper.clickOnElement(CdfPluginPropertiesLocators.locateSourcePluginNameInList(pluginName));
+  }
+
+  /**
+   * Click on Deploy the Replication pipeline
+   */
+  public static void deployReplicationPipeline() {
+      ElementHelper.clickOnElement(CdfPluginPropertiesLocators.deployReplicationPipeline);
+    }
+
+  /**
+   * Click on Start for the replication pipeline to start
+   */
+  public static void startReplicationPipeline() {
+    ElementHelper.clickIfDisplayed(CdfPluginPropertiesLocators.start, ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
+  }
+
+  /**
+   * Check for the pipeline status is running
```

Comment:
```Check for the pipeline status is running``` You are starting the pipeline here as well update the java doc

Reply:
Change done.